tests/resources/reporting_round.csv

```diff
@@ -4,12 +4,12 @@ bb265da3-999e-4282-aa2c-0fa85da503a1,2,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2022
 de52f699-f26f-44ff-b7ca-5cc834300500,3,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2022-10-01 00:00:00,2023-03-31 23:59:59,,
 eede818b-8697-4034-a4a2-7c35dff080bb,4,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2023-04-01 00:00:00,2023-09-30 23:59:59,,2023-12-04 23:59:59
 4428f656-5997-42c2-9149-b2303a6f1765,5,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2023-10-01 00:00:00,2024-03-31 23:59:59,,2024-05-28 23:59:59
-4428f654-5997-42c2-9149-b2303a6f1765,6,4a6e9f7d-fc9d-4c12-b1b6-89e784c310e1,2024-04-01 00:00:00,2024-09-30 23:59:59,,
+52062ae1-e947-4fc4-9ce1-be5d7887f42f,6,0fe2cf84-6b3c-4dda-96ec-632c76c6dc05,2024-04-01 00:00:00,2024-09-30 23:59:59,2024-10-21 00:00:00,2024-11-24 23:59:59
 311b7d4b-3545-48fc-bc9a-b2fee38ccd68,1,9fde58b2-8a89-4b2c-af7d-1f968b03c7b9,2019-04-01 00:00:00,2022-03-31 23:59:59,,
 405dc124-06b0-497c-a45a-a0947045fb10,2,9fde58b2-8a89-4b2c-af7d-1f968b03c7b9,2022-04-01 00:00:00,2022-09-30 23:59:59,,
 c9e10f59-419e-4da2-a92b-2f9f9a09923d,3,9fde58b2-8a89-4b2c-af7d-1f968b03c7b9,2022-10-01 00:00:00,2023-03-31 23:59:59,,
 49219b20-3762-4fd8-a591-53508c68b2dd,4,9fde58b2-8a89-4b2c-af7d-1f968b03c7b9,2023-04-01 00:00:00,2023-09-30 23:59:59,,2023-12-04 23:59:59
 0af7ee1e-6c97-4a12-8f4d-59158a9888d4,5,9fde58b2-8a89-4b2c-af7d-1f968b03c7b9,2023-10-01 00:00:00,2024-03-31 23:59:59,,2024-05-28 23:59:59
-4428f345-5997-42c2-9149-b2303a6f1765,6,9fde58b2-8a89-4b2c-af7d-1f968b03c7b9,2024-04-01 00:00:00,2024-09-30 23:59:59,,
+ff99f14b-8078-4e5b-99a2-475dc4aca51e,6,d57b28fb-a4f5-465c-ad51-3c7fe55669cb,2024-04-01 00:00:00,2024-09-30 23:59:59,2024-10-21 00:00:00,2024-11-24 23:59:59

```

Comment:
Please have another look at and think about these changes - currently you've changed a bit too much so these are broken.